opds.py

```diff
@@ -62,6 +62,10 @@ def default_lane_url(cls):
     def top_level_title(self):
         return "All Books"
 
+    def facet_url(self, facets):
```

Comment:
I don't understand how this works for static feeds. Aren't the facet urls something like base_feed_author.opds instead of a feed url with query parameters like in the app?